_includes/body-landing.html

```diff
@@ -0,0 +1,844 @@
+<body ng-app="Docker" ng-controller="DockerController" id="landing">
+  <nav class="navbar header">
+    <div class="container">
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#header-content" aria-expanded="false">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a class="navbar-brand" href="/">
+          <img
+            src="../images/docker-docs-logo.svg"
```

Comment:
Might be safer to use an absolute path for this. Given that this isn an "include" it could potentially be included in different paths, in which case the link would break

(either that, or set the logo through CSS, but that's more of a "nice to have")

Reply:
About the logo through CSS, considering we are using bootstrap, I'm following the examples as much as possible to avoid fighting the framework 🤷

Reply:
Yeah, the site / theme has grown to become a bit of a mish-mash; using CSS is definitely not an urgent thing (bigger mess to clean up than that)